Shared/SentryIntegration.swift

```diff
@@ -90,6 +90,12 @@ public class SentryIntegration: SentryProtocol {
 
                 return event
             }
+            options.beforeBreadcrumb = { crumb in
+                if crumb.type == "http" || crumb.category == "http" {
```

Comment:
I'm guessing this will also take care of https

Reply:
Correct